guides/v2.2/cloud/cdn/fastly-waf-service.md

```diff
@@ -57,15 +55,15 @@ Magento and Fastly manage the update process to ensure that new or modified WAF
 
 ### Limitations
 
-The standard Magento Cloud WAF powered by Fastly does not support the following features:
+The standard WAF service powered by Fastly does not support the following features:
 
-- Filtering for TCP, UDP, or ICMP requests
-- Protection against malware or bot mitigation
-- Rate limiting or virtual patching
-- Configuring a logging endpoint for customer
+-  Protection against malware or bot mitigation
+-  Rate limiting
+-  Configuring a logging endpoint for customer
 
 Although the WAF does not allow you to block or allow traffic based on IP addresses, you can add access control lists (ACL) and custom VCL snippets to your Fastly service to specify the IP addresses and VCL logic for blocking or allowing traffic. See [Custom Fastly VCL snippets]({{ page.baseurl }}/cloud/cdn/cloud-vcl-custom-snippets.html).
 
+Filtering for TCP, UDP, or ICMP requests is not supported by the WAF. However, this functionality is provided by the built-in DDoS protection included with the Fastly CDN service. See [DDoS protection]({{ page.baseurl }}/cloud/cdn/cloud-fastly.html#ddos-protection).
```

Comment:
Should all instances of WAF be `WAF service` ? If so, there are few sprinkled about that need service after it.